newIDE/app/public/external/piskel/piskel-main.js

```diff
@@ -281,87 +153,193 @@ function piskelCreateAnimation(pskl, piskelOptions) {
   pskl.app.settingsController.settingsContainer
     .getElementsByClassName('textfield resize-size-field')[0]
     .focus();
-  updatePiskelBasePath(); //update the path label
-}
+};
 
 ipcRenderer.on('piskel-load-animation', (event, receivedOptions) => {
-  piskelOptions = receivedOptions;
-  piskelAnimationNameInput.value = piskelOptions.name;
+  /**
+   * Inject custom buttons in Piskel's header,
+   * get rid of the new file button,
+   * make animation name and path editable
+   */
+  const editorContentDocument = document.getElementById('piskel-frame')
+    .contentDocument;
+  const newButton = editorContentDocument.getElementsByClassName(
+    'new-piskel-desktop button button-primary'
+  )[0];
+  const oldPiskelNameLabel = editorContentDocument.getElementsByClassName(
+    'piskel-name'
+  )[0];
+  newButton.style.display = 'none';
+  oldPiskelNameLabel.style.display = 'none';
+  const piskelAppHeader = editorContentDocument.getElementsByClassName(
+    'fake-piskelapp-header'
+  )[0];
+  piskelAppHeader.style.display = 'none';
+
+  // Load a custom save file(s) header
+  const pathEditorHeaderDiv = document.getElementById('path-editor-header');
+  const headerStyle = {
+    saveFolderLabel: 'float: left;margin-left: 2px; font-size:15px;margin-top: 10px;color:aqua',
+    nameInput: 'font-family:"Courier New";height:27px;width:90px;float:left;margin-left: 2px;padding:4px;margin-top: 4px;font-size:15px;border: 2px solid #e5cd50;border-radius: 3px;background-color:black; color: #e5cd50;',
+    saveButton: 'float:right;margin-left:2px;margin-right:4px;border: 2px solid white;border-radius: 1px;margin-top: 5px;background-color:white;',
+    cancelButton: 'float:right;margin-right:2px;border: 2px solid white;border-radius: 1px;margin-top: 5px;background-color:white;',
+    setFolderButton: 'float:right;margin-left:2px;margin-right:4px;border: 2px solid white;border-radius: 1px;margin-top: 5px;background-color:white;',
+    fileExistsLabel: 'height:27px;color:aqua;float: left;margin-left: 2px;margin-top: 10px; font-size:15px;',
+  };
+  const savePathEditor = createPathEditorHeader({
+    parentElement: pathEditorHeaderDiv,
+    editorContentDocument: document,
+    onSaveToGd: saveToGD,
+    onCancelChanges: cancelChanges,
+    projectPath: receivedOptions.projectPath,
+    initialResourcePath: receivedOptions.resources[0] === undefined ?
+      '' : receivedOptions.resources[0].resourcePath,
+    extension: '',
+    headerStyle,
+    name: receivedOptions.name,
+  });
 
+  // Get controller again
+  piskelOptions = receivedOptions;
   const editorFrameEl = document.querySelector('#piskel-frame');
-  const pskl = editorFrameEl.contentWindow.pskl;
-  if (!pskl) return;
+  pskl = editorFrameEl.contentWindow.pskl;
```

Comment:
Do we still need this at this point?

Reply:
ah you're right. I forgot to remove it sorry